src/siteconfig/migrations/0030_siteconfig_allow_staff_export.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.21 on 2025-08-01 18:10
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('siteconfig', '0029_siteconfig_map_auto_update'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='siteconfig',
+            name='allow_staff_export',
+            field=models.BooleanField(default=False, help_text='If enabled, staff will be able to export quests and campaigns to the Shared Library.', verbose_name='Allow staff to export quests to the library'),
```

Comment:
Not clear the status when off.  So maybe change to: "If enabled, staff will be able to export quests and campaigns to the Shared Library, otherwise only the owner can export quests to the Library"

 (if you change this please squash the migrations so we only have one, or delete this migration so it regenerates it fresh... stupid that it generates a migration when helptext is changed)

Reply:
Can we hyperlink this helptext to the Library help/wiki page?